cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -746,6 +746,7 @@ func (command *Evaluate) evaluateKubernetes(ctx *evaluate.Context) (err error) {
 				Command: []string{
 					"kubectl",
 					"wait",
+					"--timeout", "24h",
```

Comment:
Is that the whole evaluation or just a single container? I think a single container might be faster... But what if the full evaluation is longer? Would be a bit sad for someone running a huge eval and then it gets killed.

Reply:
It is a single container